src/App.css

```diff
@@ -80,39 +79,39 @@ span.empty {
 
 .call-empty {
   border: 1px solid #eee;
-  background: #fff; }
+  background: white; }
```

Comment:
This sounds odd, but it's considered bad practice to use named colors in the default space. Please make a variable called `$white: #fff;`